webapp/src/DOMJudgeBundle/Controller/API/AbstractRestController.php

```diff
@@ -165,7 +165,7 @@ protected function getContestQueryBuilder(): QueryBuilder
         if (!$this->dj->checkrole('api_reader') && !$this->dj->checkrole('judgehost')) {
             if ($this->dj->checkrole('team') && $this->dj->getUser()->getTeamid()) {
                 $qb->leftJoin('c.teams', 'ct')
-                    ->andWhere('ct.teamid = :teamid OR c.public = 1')
+                    ->andWhere('ct.teamid = :teamid OR c.allTeams = 1')
```

Comment:
As above: `all_teams`?

Reply:
This one uses the doctrine entity so I think this one is correct. The above one should be fixed yes

Reply:
Hmmm, ok. I would still argue to write it using the SQL column name for consistency and (from my point of view) less confusion.

Reply:
Well my IDE autocompletes to this way of writing it currently. I think it even breaks if I use the SQL column name, because that field doesn't exist on the entity.

We could rename the property in the entity from `openToAllTeams` to `open_to_all_teams`, then it does what you want. Shall I do that?